mmpose/datasets/pipelines/pose3d_transform.py

```diff
@@ -235,27 +366,42 @@ def __call__(self, results):
         if np.random.rand() <= self.flip_prob:
 
             flip_pairs = results['ann_info']['flip_pairs']
+
             # flip joint coordinates
-            assert self.item in results
-            joints = results[self.item]
+            for i, item in enumerate(self.item):
+                assert item in results
+                joints = results[item]
 
-            joints_flipped = fliplr_regression(
-                joints,
-                flip_pairs,
-                center_mode='root',
-                center_index=self.root_index)
+                joints_flipped = fliplr_regression(joints, flip_pairs,
+                                                   **self.flip_cfg[i])
 
-            results[self.item] = joints_flipped
+                results[item] = joints_flipped
 
             # flip joint visibility
-            if self.vis_item is not None:
-                assert self.vis_item in results
-                visible = results[self.vis_item]
+            for i, vis_item in enumerate(self.vis_item):
+                assert vis_item in results
+                visible = results[vis_item]
                 visible_flipped = visible.copy()
                 for left, right in flip_pairs:
                     visible_flipped[..., left, :] = visible[..., right, :]
                     visible_flipped[..., right, :] = visible[..., left, :]
-                results[self.vis_item] = visible_flipped
+                results[vis_item] = visible_flipped
+
+            # flip horizontal distortion coefficients
+            if self.flip_camera:
+                if self.static_camera:
+                    camera_param = self.camera_param
+                else:
+                    assert 'camera_param' in results, \
+                        'Camera parameters are missing.'
+                    camera_param = copy.deepcopy(results['camera_param'])
+                assert 'c' in camera_param and 'p' in camera_param
```

Comment:
'p' should be an optional parameter so we just flip it when it is contained in camera_param.